components/ActionSheet/tests/unit/MDCActionSheetTest.m

```diff
@@ -204,4 +218,142 @@ - (void)testSetNilTitleAndMessageColor {
                         [UIColor.blackColor colorWithAlphaComponent:0.6f]);
 }
 
+#pragma mark - Opening height
+
+- (void)addActions:(NSUInteger)actions {
+  for (NSUInteger i = 0; i < actions; ++i) {
```

Comment:
Tip: `ix` instead of `i`, or `actionIndex` to be most ideal.

Reply:
Done